packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -115,7 +115,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
           menuIconProps && menuIconProps.className,
           isPrimaryButtonDisabled!,
           checked!,
-          this._isMenuExpanded(),
+          this._isExpanded,
```

Comment:
This should call the function since `getClassNames` expects a boolean, so you're missing the parentheses `()` there.

Reply:
`_isExpanded` is a getter. It should works as is

Reply:
Sorry, I don't know how this went over my head.